plugins/modules/inventory_workflow_manager.py

```diff
@@ -1755,9 +1768,9 @@ def get_site_type(self, site_name):
                     site_type = item.get("attributes").get("type")
 
         except Exception as e:
-            self.msg = "Error while fetching the site '{0}'.".format(site_name)
+            self.msg = "Error while fetching the site '{0}' and given site not found in Cisco Catalyst Center".format(site_name)
```

Comment:
and the specified site was not found in Cisco Catalyst Center.

Reply:
added.

Reply:
added.